dev-libs/libclc/libclc-18.1.4.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=8
 
-LLVM_COMPAT=( 17 )
+LLVM_COMPAT=( 18 )
```

Comment:
Make it `17..18`. The only reasons older versions ain't here are limited keywords on spirv-llvm-translator.